controllers/routes.js

```diff
@@ -133,12 +133,25 @@ let funding = {
 // Add Funding subpages in non-prod environments
 // @TODO progress to production when ready
 if (isNotProduction) {
-    funding.pages.push({
-        path: '/*',
-        router: basicContent({
-            cmsPage: true
-        })
-    });
+    funding.pages.push(
+        {
+            path: `/managing-your-grant/promoting-your-project/order-free-plaques-stickers-bunting-and-more`,
+            router: require('./materials')
+        },
+        {
+            path: `/managing-your-grant/promoting-your-project/download-our-logo-to-tell-people-about-your-national-lottery-funding`,
```

Comment:
Do we need to add some redirects for the old locations? Also, worth using this as an excuse to simplify the urls e.g. `managing-your-grant/promoting-your-project/logos`

Reply:
I was planning to do this when we put these new pages live (soon, this week perhaps) rather than add more conditional checks to the code. Could change the URLs though, will ask Scott!